core/src/main/scala/org/apache/spark/scheduler/AsyncEventQueue.scala

```diff
@@ -169,7 +169,8 @@ private class AsyncEventQueue(
           val prevLastReportTimestamp = lastReportTimestamp
           lastReportTimestamp = System.currentTimeMillis()
           val previous = new java.util.Date(prevLastReportTimestamp)
-          logWarning(s"Dropped $droppedCount events from $name since $previous.")
+          logWarning(s"Dropped $droppedCount events from $name since " +
+            s"${if (prevLastReportTimestamp == 0) "the application starts" else s"$previous"}.")
```

Comment:
Actually the previous version was grammatically correct. Either:

- since the application started
- since the start of the application

But not the current version...

Reply:
@vanzin 
Thanks for your comment.
Updated code according to your suggestion.

Reply:
Oops. I missed the past tense.